### ISSUE TITLE
question about confidence interval for INDELs

### ISSUE THREAD

**Issue by loipf** [im3sanger/dndscv#91]
Hi,
for SNVs, one can easily calculate confidence intervals for the dN/dS ratios per gene using the `geneci()`, which is really useful. However, it only calculates confidence intervals for SNV classes. Is there an easy way to also calculate confidence intervals for INDELs with observed/expected ratios or does it need a different underlying statistical model?
Thanks in advance.

**Comment by 676lql** [comment]
Hi，I have the same question as you, have you solved your problem yet?

**Comment by loipf** [comment]
No, but I found a paper that implemented confidence intervals for indels and even came up with a statistical test to compare which genes are different under selective pressure between two populations. Unfortunately, they do not share their code and I am not able to re-implement it.

Zhu, Bin et al. “Comparison of somatic mutation landscapes in Chinese versus European breast cancer patients.” HGG advances vol. 3,1 100076. 3 Dec. 2021, doi:10.1016/j.xhgg.2021.100076
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8756551/
